core/insert-pack.cpp

```diff
@@ -64,4 +65,23 @@ void SnowboardInsertPack::addInsertGroup(MCFixed stanceX) {
   m_insertsPath.push_back(Point(stanceX + x, -y));
 }
 
+//------------------------------------------------------------------------------
+// Ski insert pack (also used for any generic binding pattern)
+
+SkiInsertPack::SkiInsertPack(const std::vector<double>& insertX, 
+                             const std::vector<double>& insertY)
+    //: m_insertX(insertX[])
+    //, m_insertY(insertY[])
+{
+ for (unsigned it = 0; it != insertX.size(); it++) {
```

Comment:
There's a bit of wonky indents and extra whitespace here and there. If you could clean it up a bit that'd be great!